chia/plotting/create_plots.py

```diff
@@ -42,10 +42,10 @@ def pool_contract_puzzle_hash(self) -> Optional[bytes32]:
 class PlotKeysResolver:
     def __init__(
         self,
-        farmer_public_key: str,
-        alt_fingerprint: int,
-        pool_public_key: str,
-        pool_contract_address: str,
+        farmer_public_key: Optional[str],
+        alt_fingerprint: Optional[int],
+        pool_public_key: Optional[str],
+        pool_contract_address: Optional[str],
```

Comment:
Why is this change included? I thought this was about adding `__init__.py` files?

Reply:
Adding `chia/plotting/__init__.py` enabled more `mypy` checks in the related code.